packages/showcase/src/app/components/drop-down/metadata.ts

```diff
@@ -0,0 +1,138 @@
+import {
+  nestedModel,
+  plainModel,
+  withSeparatorsModel,
+  withSlotModel,
+  withIconsModelForMetadata
+} from "./models";
+
+// Common
+const DROP_DOWN_TAG = "ch-action-menu-render";
+const DROP_DOWN_CLASS = "dropdown";
+const CHILDREN = "Open Menu";
+const PROPERTIES = [
+  { name: "model", value: "actionMenuModel", state: true },
+  { name: "expanded", value: false, variable: true },
+  { name: "inlineAlign", value: "inside-start" },
+  { name: "blockAlign", value: "outside-end" }
+];
+const IMPORTS_NO_ICONS = [chameleonImportType("ActionMenuModel")];
+
+export const dropDownMetadata = {
+  title: "Drop Down",
+  description:
+    "The dropdown component is an interactive element used to display a list of options from which users can select. It supports various states (open, disabled, focused) and styles (standard, outlined, filled) for customization and accessibility.",
+  codeSnippets: {
+    plain: {
+      linkId: "plain",
+      title: "1. Plain",
+      imports: IMPORTS_NO_ICONS,
+      states: [
+        {
+          name: "actionMenuModel",
+          type: "ActionMenuModel",
+          value: plainModel
+        }
+      ],
+      template: {
+        tag: DROP_DOWN_TAG,
+        class: DROP_DOWN_CLASS,
+        properties: PROPERTIES,
+        children: CHILDREN
+      }
+    },
+
+    nested: {
+      linkId: "nested",
+      title: "2. Nested",
+      imports: IMPORTS_NO_ICONS,
+      states: [
+        {
+          name: "actionMenuModel",
+          type: "ActionMenuModel",
+          value: nestedModel
+        }
+      ],
+      template: {
+        tag: DROP_DOWN_TAG,
+        class: DROP_DOWN_CLASS,
+        properties: PROPERTIES,
+        children: CHILDREN
+      }
+    },
+
+    withSeparators: {
+      linkId: "with-separators",
+      title: "3. With Separators",
+      imports: IMPORTS_NO_ICONS,
+      states: [
+        {
+          name: "actionMenuModel",
+          type: "ActionMenuModel",
+          value: withSeparatorsModel
+        }
+      ],
+      template: {
+        tag: DROP_DOWN_TAG,
+        class: DROP_DOWN_CLASS,
+        properties: PROPERTIES,
+        children: CHILDREN
+      }
+    },
+
+    withSlots: {
+      linkId: "with-slots",
+      title: "4. With Slots",
+      imports: IMPORTS_NO_ICONS,
+      states: [
+        {
+          name: "actionMenuModel",
+          type: "ActionMenuModel",
+          value: withSlotModel
+        }
+      ],
+      template: {
+        tag: DROP_DOWN_TAG,
+        class: DROP_DOWN_CLASS,
+        properties: PROPERTIES,
+        children: [
+          {
+            tag: "p",
+            properties: [{ name: "slot", value: "item-header" }],
+            children: "This is the header slot"
+          },
+          "Open Menu",
+          {
+            tag: "p",
+            properties: [{ name: "slot", value: "item-footer" }],
+            children: "This is the footer slot"
+          }
```

Comment:
It would be nice the add a better example for this feature. For example, the user menu case:
https://www.figma.com/design/o96qnWUmbfu2dsPuCPjcVI/Conversational-Layouts?node-id=16-5063&t=nO5eGWVtXzJiZptb-0